base/leaky_bucket.go

```diff
@@ -649,6 +649,10 @@ func (b *LeakyBucket) IsError(err error, errorType sgbucket.DataStoreErrorType)
 	return b.bucket.IsError(err, errorType)
 }
 
+func (b *LeakyBucket) GetExpiry(k string) (expiry uint32, err error) {
+	return b.GetUnderlyingBucket().GetExpiry(k)
```

Comment:
nit: looks like the other methods use `b.bucket.Foo()` rather than `b.GetUnderlyingBucket().Foo()` - any particular reason for not doing the same here?